docs/sharing/design-integrations.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Design integrations'
+---
+
+Storybook supports first-class integrations with well-known design tools, speeding up the development cycle by enabling you to cross-reference designs with your existing stories.
+
+![Storybook design integrations](./storybook-design-integrations.jpeg)
+
+For development teams, Storybook provides the necessary tools to build, [test](../writing-tests/introduction.md), and [document](../writing-docs/introduction.md) UI components. [Integrating](https://storybook.js.org/addons/tag/design/) your Storybook with popular design tools (for example, [Figma](https://www.figma.com/), [Zeplin](https://zeplin.io/)) unlocks new workflows for you and your team. You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.
```

Comment:
> You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.

There's a missing word here, I think.

Reply:
Don't think so. I double-checked it and it panned out!